pages/application-complete.js

```diff
@@ -1,7 +1,6 @@
 import SessionWrapper from "../components/SessionWrapper";
 import FsaLayout from "../components/FsaLayout";
-import { Header, Panel } from "govuk-react";
-import Paragraph from "@govuk-react/paragraph";
+import { Header, Panel, Paragraph } from "govuk-react";
```

Comment:
Considering we're now importing Paragraph from the main library, can we run `yarn remove @govuk-react/paragraph` ?